### ISSUE TITLE
Trigger settings have no effect

### ISSUE THREAD

**Issue by olafgrabienski** [backdrop-contrib/beautytips_node#6]
On the configuration page `admin/config/content/beautytips/node-settings` there are options to select various triggers for *BeautyTip On* and *BeautyTip Off*,  for instance `hover`, `hoverIntent`, `click`, or `dblclick`.

When I change the `hover` defaults to other triggers but the setting has no effect: tooltip links are only triggered by `hover`.

In https://github.com/backdrop-contrib/beautytips_node/blob/1.x-1.x/js/beautytips_node.js I didn't find any reference to different trigger settings, so I had a look at the corresponding file of the Drupal module (https://github.com/backdrop-contrib/beautytips_node/blob/7.x-1.x/js/beautytips_node.js) where there are such references, e.g. around line 25, but they're commented out. So, if I understand it correctly, the trigger settings didn't work in Drupal as well.

Do we want to drop support for the trigger settings, or do we want to (re-)introduce it? For various reasons I'd prefer to try the latter. @Graham-72 what do you think about it?

**Comment by Graham-72** [comment]
Yes, it would seem that the Drupal project beautytips_node is unfinished and so more work is needed, perhaps working towards a release 1.x-1.1.0 in Backdrop. @olafgrabienski I have no plans to use the module myself but if you would like to identify any particular features you have a definite use for, I am willing to see if I can add them.

**Comment by olafgrabienski** [comment]
@Graham-72, thanks! Indeed I'd like to use the trigger settings, or at least I think so. My use case: People want to be able to click links in a tooltip. That doesn't work with the `hover` trigger but *maybe* with `hoverIntent` (tooltips don't disappear immediately when the mouse moves away from the link) or one of the other triggers. I'm however not sure if changing the trigger is the right approach.

(Apart of that, experimenting with the trigger settings could help to find out how to add keyboard support for *Beautytips Node*. I noticed that it's not possible to open a *Beautytips node* tooltip using the keyboard instead of the mouse.)

So, all in all it would be nice to add the trigger settings. However, if it's too difficult, or too much work, I guess it makes more sense that I look for a more direct solution for my use case.
